syntax = "proto3";

package seerep;

message ProjectQuery
{
  // uuid of the project to query
  string projectuuid = 1;
}
